lib/runner.js

```diff
@@ -538,7 +548,13 @@ Runner.prototype.runTests = function (suite, fn) {
     self.emit('test', self.test = test);
     self.hookDown('beforeEach', function (err, errSuite) {
       if (test.isPending()) {
-        self.emit('pending', test);
+        if (self.forbidPending) {
+          test.isPending = alwaysFalse;
+          self.fail(test, new Error("Pending test forbidden"));
+          delete test.isPending;
+        } else {
+          self.emit('pending', test);
+        }
         self.emit('test end', test);
         return next();
       }
```

Comment:
These two `if (test.isPending()) { ...` sections are identical or nearly so; we should consider breaking it out into a function or something.